client/src/GamemasterInGame/CharacterSelectList.tsx

```diff
@@ -76,37 +79,62 @@ export function CharacterSelectList({
     }));
     setNewCharacterName("");
   }
+  const charactersByType: Record<CharacterType, Character[]> = useMemo(() => {
+    const allCharacters = [
+      ...state.characters,
+      ...state.additionalCharacters.value,
+    ];
+    return {
+      Townsfolk: allCharacters.filter((c) => c.team === "Townsfolk"),
+      Outsider: allCharacters.filter((c) => c.team === "Outsider"),
+      Minion: allCharacters.filter((c) => c.team === "Minion"),
+      Demon: allCharacters.filter((c) => c.team === "Demon"),
```

Comment:
Maybe we should put an "Unknown" in here for any imported unrecognized characters?

Reply:
Right now unknowns are `team: ""` but that can just as easily be changed to something more helpful.